server.js

```diff
@@ -21,6 +23,10 @@ const handle = app.getRequestHandler();
 
 dotenv.config();
 
+Sentry.init({
+  dsn: "https://b8325acff40e4d548871a23835ce9163@sentry.io/1800052"
```

Comment:
not sure how sentry works, is this a secret/should it be in `.env` ?

Reply:
Good question; I'll double check but it didn't seem like it?

Reply:
my brief google search says it shouldn't be a problem, LGTM (client-side error stuff would be hella cool, but not super pressing)